manifests/install.pp

```diff
@@ -208,11 +208,6 @@
     ensure => running,
     subscribe => [File["/usr/share/jetty9/webapps/blazegraph/WEB-INF/GraphStore.properties"]],
 } ->
-exec { "wait_for_blazegrah":
```

Comment:
Why was this removed?

Reply:
It didn't work

Reply:
So the previous ensure is good enough?

The for might need bash -c '' around it, to start a shell, if we still need the curl tries.